README.md

```diff
@@ -874,6 +874,47 @@ $authenticationRequest = new UsernamePasswordRequest('usernameOrEmail', 'passwor
 $result = $application->authenticateAccount($authenticationRequest);
 ```
 
+### Verify an Account's email address
+
+This workflow allows you to send a welcome email to a newly registered account and optionally verify that they own the 
+email addressed used during registration.
+
+The email verification workflow involves changes to an account at an application level, and as such, this workflow
+relies on the account resource as a starting point. This workflow is disabled by default for accounts, but you can 
+enable it easily in the Stormpath Admin Console UI. Refer to the Stormpath Admin Console product guide for complete
+instructions.
```

Comment:
We should be able to do this from the REST API, not only through the UI: http://docs.stormpath.com/rest/product-guide/#directory-account-creation-policy